config/lava/chromeos/chromiumosvm-tast-template.jinja2

```diff
@@ -0,0 +1,42 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{{ super () }}
+{% endblock %}
+
+{% block actions %}
+{{ super () }}
+
+actions:
+- deploy:
+    images:
+      bios:
+        image_arg: -bios {bios}
+        url: http://storage.kernelci.org/images/uefi/edk2-stable202005/x86_64/OVMF.fd
+      rootfs:
+        image_arg: -drive format=raw,file={rootfs},id=lavatest2,if=none
+        url: {{ rootfs_url }}chromiumos_test_image.bin
+        compression: gz
+{%- if kernel_url %}
+#       kernel:
+#         image_arg: -kernel {kernel} -append "init=/sbin/init boot=local rootwait ro noresume noswap loglevel=7 earlyprintk=serial,keep console=tty0 console=ttyS0 i915.modeset=1 cros_efi cros_debug root=/dev/sda3"
+#         url: {{ kernel_url }}
```

Comment:
We probably should resolve this somehow.  I would suggest:
* get some Chromium OS tests on QEMU scheduled by KernelCI
* get the results on chromeos.kernelci.org web dashboard to confirm it's all working
* un-comment this part to enable the KernelCI kernels to be used (linux-next) and see if some config works with QEMU
* if the KenrelCI kernels don't work then add a patch to comment out this section, so we can drop it later

Reply:
I will add tast tarball generation in builder script.

Reply:
If it's failing all the tests and they're reported as failures then that's good enough for KernelCI. But I think we should go through each step mentioned above unless you want to suggest something else.

Reply:
About extra patches, that's something to discuss with Google folks and others. Ideally we shouldn't be applying anything as KernelCI is to test upstream, and report all the things that aren't working.

Reply:
I uncommented it and i will schedule tests after also i change build script to generate tast tgz

Reply:
But how about getting results in KernelCI first before trying with real kernel builds?

I'm trying to help get this resolved hence the list of steps I'm suggesting but I don't understand the approach of trying with KernelCI build now as we haven't proved we can get results with a working kernel all the way to the web dashboard. I don't think the jobs are even submitted yet, so that's something to check and fix first.

Reply:
I mean i will try to build rootfs image with tast tarball changes first, then i will generate LAVA definition manually (by kci_test) with upstream kernel and will check how it goes and what results we get on LAVA.